renovate.json

```diff
@@ -2,6 +2,7 @@
   "extends": [
     "config:base"
   ],
+  "updateInternalDeps": true,
```

Comment:
According to the Renovate docs, this _should_ tell Renovate to update all the dependencies listed in `packages/**/package.json` files in addition to the `package.json` file in the repository root.